CHANGELOG.md

```diff
@@ -2,6 +2,128 @@
 
 About the past changelog entries, see [old CHANGELOG](CHANGELOG-v4.md) instead.
 
+## Release v5.0.7 - 2025/05/14
+
+### News
+
+* Update Ruby to 3.2.8 (#816)
+* Update fluentd to 1.16.9 (#816)
+* Update bundled gems
+
+### Core component
+
+* ruby v3.2.8 (update)
+* jemalloc v3.6.0
+* OpenSSL 3.4.0 Windows
+* OpenSSL 3.0.8 macOS
+* fluentd v1.16.9 (update)
+* msgpack 1.7.5
+* oj 3.16.5
+* webrick 1.8.2
+* openssl 3.3.0 (update)
```

Comment:
```suggestion
* gems
  * fluentd v1.16.9 (update)
  * msgpack 1.7.5
  * oj 3.16.5
  * webrick 1.8.2
  * openssl 3.3.0 (update)
```

Because they are confusing: `OpenSSL` and `openssl`